app/__init__.py

```diff
@@ -74,6 +74,9 @@ def create_app(application):
     application.config.from_object(configs[notify_environment])
 
     application.config["NOTIFY_APP_NAME"] = application.name
```

Comment:
Separately, I think this is a really weird thing we're setting and I might look at updating this. I think it's always just "app" 🙃